tests/gnrc_ipv6_nib_dns/Makefile

```diff
@@ -29,8 +29,8 @@ LOW_MEMORY_BOARDS := nucleo-f334r8 msb-430 msb-430h
 
 ifeq ($(BOARD),$(filter $(BOARD),$(LOW_MEMORY_BOARDS)))
     CFLAGS += -DGNRC_PKTBUF_SIZE=512 -DCONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF=2 \
-              -DGNRC_NETIF_IPV6_GROUPS_NUMOF=2 -DGNRC_IPV6_NIB_NUMOF=1 \
-              -DNRC_IPV6_NIB_OFFL_NUMOF=1
+              -DGNRC_NETIF_IPV6_GROUPS_NUMOF=2 -DCONFIG_GNRC_IPV6_NIB_NUMOF=1 \
```

Comment:
Since `CONFIG_GNRC_IPV6_NIB_NUMOF` defaults to 1 when `CONFIG_GNRC_IPV6_NIB_6LR` is not set, this might not even be necessary.

Reply:
Hm that might be true. Nonetheless, it has a nice documentational character as part of a test application. I have no strong feelings about it, though.

Reply:
Also: only because it is the default _now_, it might not be tomorrow. The test assumes appearently `NIB_NUMOF==1` to function, so removing it might break the test later on.